app.js

```diff
@@ -248,6 +262,7 @@ function createChart() {
 }
 
 //variables needed for this to work
+var pictureStoredData;
 var totalNumberOfClicks = 0;
 var picturesOnScreenIndex = [];
 var pictureBox = document.getElementById('pictureArrays');
```

Comment:
Best practice is to declare your global variables at the top of your file preceding your function definitions.